scripts/release_scripts/release_info.py

```diff
@@ -28,8 +28,7 @@
 
```

Comment:
Can we rename this script to generate_release_info.py? (Similar to other scripts which start with a verb.)

Reply:
Done.